src/main/java/net/dv8tion/jda/core/EmbedBuilder.java

```diff
@@ -238,6 +258,18 @@ public EmbedBuilder appendDescription(CharSequence description)
     }
 
     /**
+     * The description of the embed, can be set using {@link EmbedBuilder#setDescription(CharSequence)}
+     * or appended using {@link EmbedBuilder#appendDescription(CharSequence)}.
+     *
+     * @return Possibly-null String containing a description of the embed.
+     */
+    public String getDescription()
+    {
+        return description.toString();
+    }
```

Comment:
`Possibly-null String containing a description of the embed`
In this case it would never be null, however it should be null if the StringBuilder is empty.